doc/rst/language/spec/types.rst

```diff
@@ -175,30 +258,136 @@ uint(64), uint 0                    18446744073709551615
 Integer literals such as `3` have type ``int``. However, such literals
 can implicitly convert to other numeric types that can losslessly store
 the value. See :ref:`Implicit_Compile_Time_Constant_Conversions`.
+Integer literals can be written in hexadecimal, octal, or binary. See
+:ref:`Literals`.
+
+Signed or unsigned integral types of smaller width can implicitly convert
+to signed integral types of larger width.  Additionally, unsigned
+integral types of a smaller width can implicitly convert to an unsigned
+integral type of larger width. See :`Implicit_NumBool_Conversions` for
+details.
```

Comment:
It was a bit hard for me to read. How about this:

Signed integral types of can implicitly convert to signed integral types of larger width.  Unsigned integral types can implicitly convert to both signed and unsigned integral type of larger width. See ....

Also, should we say that int(64) can implicitly convert to uint(64) (or in the other direction), if that is the case?